packages/listview/src/components/FloatingSearch/search.js

```diff
@@ -49,7 +55,12 @@ const Search = forwardRef((props, ref) => {
                     </StyledClearButton>
                     <StyledDivider />
                 </RenderIf>
-                <ButtonIcon size="small" icon={<Close />} onClick={onRequestClose} />
+                <ButtonIcon
+                    id="close-search-button"
```

Comment:
I think our components should not use id internally, instead, you can use data-* like data-id because id must be unique in the document and then we can't know what id the user will use